app/controllers/application_controller.rb

```diff
@@ -797,7 +797,7 @@ def flash_error(*strs)
   def flash_object_errors(obj)
     return unless obj&.errors && !obj.errors.empty?
 
-    flash_error(obj.formatted_errors)
+    obj.formatted_errors.each { |error| flash_error(error) }
```

Comment:
Fixes a bug. `object.formatted.errors` is an array of strings, rather than a simple string.

Reply:
Good catch.  Another way to solve this would be just to add a star:

    flash_error(*obj.formatted_errors)

Although, honestly, your way is more clearly going to work.

Reply:
Thanks! I hadn't thought about adding a star. 
Per your comment I'll leave it as explicitly iterating over the object's formatted_errors